website/docs/installation.md

```diff
@@ -27,12 +31,36 @@ The easiest way to install Docusaurus is to use the command line tool that helps
 npx create-docusaurus@latest [name] [template]
 ```
 
-Example:
+To create a new Docusaurus app, you may choose one of the following package manager that you like:
+
+<Tabs>
+<TabItem value="npm/npx">
 
 ```bash
+# Modern way available since npm v6+
+npm init docusaurus website classic
```

Comment:
See https://github.com/facebook/docusaurus/pull/5797. Generally I don't think that's a problem because the user can use the interactive prompting to provide options anyways, but it's better if we can be always safe.

Reply:
I'm not a fan of documenting multiple versions across our doc, or cluttering it with too many tabs. 

"npx" is the only one that works well across node/npm versions without added complexity

Wouldn't it be odd if we use npx in some places and npm init in other places?

See for example TS: https://docusaurus.io/docs/typescript-support#initialization

`npx create-docusaurus@latest my-website classic --typescript`

This is hard to convert to npm init in a way that works for both npm6/7

Let's not change something that works. Most users will be happy with npx. 

Power users that know their package managers can still decide themselves to use `npm init` or `yarn create`,; and I'm ok to force a specific package manager in this case

Reply:
I just liked describing the possible options for initializing a new project, as was done in [the CRA docs](https://create-react-app.dev/docs/getting-started#creating-an-app). Perhaps not all users know about `yarn create`, even if they use yarn. Alright, I'll delete it, but what about the paragraph with the yarn requirement, it doesn't need to be reverted?

Reply:
I also believe some more exhaustive documentation is necessary in at least one place. The typescript support documentation is only meant to be a brief example that works, but we can't be like "be exhaustive everywhere or be exhaustive nowhere".

Reply:
what about adding tabs in a details/summary component?

I just really want to avoid cluttering the doc with info that most users don't need to see. 

Usually it's a one-shot to generate the project, it's not that important to document deeply everything and is actually harmful. Frontend devs know how to delete a lockfile and generate it again with another package manager. Non-dev users will just have npm and don't care about alternatives.

Reply:
> what about adding tabs in a details/summary component?

Oh, that's a good idea, added to the very end of the section.

> Docusaurus has a lot of non-dev users that don't even know what npm is anyway

Well I would not say so, in fact Docusaurus is also very much used by [open-source] developers.

>  Frontend devs know how to delete a lockfile and generate it again with another package manager

Yes, but a developer needs to do one unnecessary action then, it is not very user-friendly in general.